### PR TITLE
Fungi brar 2012 5

### PR DESCRIPTION
**Pull request by FlicAnderson**
Adding annotation files and contaminants, and config yaml file for Saccharomyces cerevisiae, dataset Brar 2012. 

Annotation files and contaminants files within separate folders within Saccharomyces cerevisiae folder.

Provenance information of contaminants file and annotations given in .txt files as suggested in [this comment](https://github.com/riboviz/example-datasets/issues/5#issuecomment-640465264) on issue #5 

Creating another folder for contaminants wasn't suggested in documentation, but I thought it might be helpful to do as annotations were also in a separate folder, and it didn't suggest including this with those?

Improvements welcome!

### CODE REVIEW

File: fungi/saccharomyces/annotation/Saccharomyces_cerevisiae_yeast_CDS_w_250utrs_annotation_provenance.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+These files hold S288c annotations and ORF sequences.
+
+These files were created as follows:
+
+The file genome release R64-2-1 (file name S288C_reference_genome_R64-2-1_20150113.tgz; https://downloads.yeastgenome.org/sequence/S288C_reference/genome_releases/S288C_reference_genome_R64-2-1_20150113.tgz) was downloaded from the Saccharomyces Genome Database (https://www.yeastgenome.org/).
+
+The files saccharomyces_cerevisiae_R64-2-1_20150113.gff and S288C_reference_sequence_R64-2-1_20150113.fsa were extracted from the .tgz file.
+
+The sequence and annotation files for the whole approximate Saccharomyces cerevisiae transcriptome were prepared using script_for_transcript_annotation.Rmd (https://github.com/riboviz/riboviz/blob/master/rmarkdown/script_for_transcript_annotation.Rmd).
```
Comment (mikej888) [inline fungi/saccharomyces/annotation/Saccharomyces_cerevisiae_yeast_CDS_w_250utrs_annotation_provenance.txt:17]: Add the commit hash for the version of master used (as the last commit on "master" was tagged 2.0.beta, that could be cited instead).

File: fungi/saccharomyces/contaminants/Saccharomyces_cerevisiae_yeast_rRNA_R64-1-1-fasta_provenance.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+This files was created as follows:
+
+The file genome release R64-1-1 (file name S288C_reference_genome_R64-1-1_20110203.tgz (https://downloads.yeastgenome.org/sequence/S288C_reference/genome_releases/S288C_reference_genome_R64-1-1_20110203.tgz)) was downloaded from the Saccharomyces Genome Database (https://www.yeastgenome.org/).
+The file rna_coding_R64-1-1_20110203.fasta was extracted from the .tgz file.
+Selected RDN-n-n sequences were copied and pasted from this file.
```
Comment (mikej888) [inline fungi/saccharomyces/contaminants/Saccharomyces_cerevisiae_yeast_rRNA_R64-1-1-fasta_provenance.txt:5]: State how the sequences were selected.

**Comment by ewallace** [comment]
Actually @FlicAnderson, I was suggesting putting contaminants in the annotation folder as well, as that would be shared for the same datasets as the other annotation info. Could you fix?

**Comment by ewallace** [comment]
Upon further reflection, I agree with Flic's suggestion to have contaminants in a distinct folder from annotations. That is sensible given that many different transcriptome annotations for a single genome might be run with the same contaminants file.

I've updated the overall `README.md` to make this clearer and to correctly refer to this dataset as an example.

Also disambiguated the provenance that `author` is meant to refer to the people who wrote the `config.yaml` file, not who wrote the paper. So I edited that to refer to @FlicAnderson.

This file has been run successfully by @XUEXUEXUE0 as referenced in #5. Do we need to update the `riboviz-version` field to reflect that, or can we merge and close as-is?

I'd like us to review and close this issue w/c 22nd June.

**Comment by mikej888** [comment]
> This file has been run successfully by @XUEXUEXUE0 as referenced in #5. Do we need to update the riboviz-version field to reflect that, or can we merge and close as-is?

This depends on what versions can, or should, be specified in example-datasets configuration files:

* Allowing arbitrary RiboViz versions is the most flexible - literally any version can be cited - but means that, when rerunning, third parties may use versions that were work-in-progress or changed significantly after that version and before being merged into "develop" then into a release.
* Allowing versions that have been merged into the "develop" branch means that only stable versions are cited. 
* Allowing versions that have been merged into releases means that only stable versions are cited too, but it restricts contributors - including ourselves - and third-parties rerunning the examples to use these releases only. Though, encouraging the use of official releases only can be a good thing as it limits the number of official versions to support. For Siyin's example this would mean waiting until release 2.0, then updating the configuration file.

**Comment by FlicAnderson** [comment]
Made the 'add commit hash' changes @mikej888 recommended, and then realised Edward had made changes I hadn't pulled. Think I've pulled/merged those now and all should be well.

**Pull request merged**